README.md

```diff
@@ -285,6 +287,9 @@ Table of Contents
   * http://www.dreamfactory.com/ - DreamFactory is an open source backend platform that provides all of the RESTful services you need to build fantastic mobile and web applications
   * https://www.mashape.com/ - API Marketplace And Powerful Tools For Private And Public APIs. With the free tier, some features are limited such as monitoring, alerting and support.
   * https://onesignal.com/ - Unlimited free push notifications
+  * https://tyk.io/ - API management with authentication, quotas, monitoring, and analytics. Free cloud offering.
+  * http://www.iron.io/ - Async task processing (like AWS Lambda) with free tier and 1 month free trial.
+  * http://stackhut.com/ - Async task processing (like AWS Lambda). Free while in beta.
```

Comment:
free-while-in-beta arent ones I'd like for this list - unless they also state they will keep a free tier post beta which doesnt seem to be the case here